mindsdb/interfaces/storage/db.py

```diff
@@ -334,3 +334,38 @@ class ChatBotsHistory(Base):
     destination = Column(String)
     sent_at = Column(DateTime, default=datetime.datetime.now)
     error = Column(String)
+
+
+class Triggers(Base):
+    __tablename__ = 'triggers'
+    id = Column(Integer, primary_key=True)
+
+    name = Column(String, nullable=False)
+    project_id = Column(Integer, nullable=False)
+
+    database_id = Column(Integer)
+    table_name = Column(String, nullable=False)
+    query_str = Column(String, nullable=False)
+    columns = Column(String, nullable=False)  # list of columns separated by delimiter
+
+    created_at = Column(DateTime, default=datetime.datetime.now)
+
+
+class Tasks(Base):
+    __tablename__ = 'tasks'
+    id = Column(Integer, primary_key=True)
+    company_id = Column(Integer)
+    user_class = Column(Integer, nullable=True)
+
+    # trigger, chatbot
+    object_type = Column(String, nullable=False)
+    object_id = Column(Integer, nullable=False)
+
+    last_error = Column(String)
```

Comment:
Looks like data in table could be updated. In that case let add `updated_at` field

Reply:
may be also add to `triggers`

Reply:
done in chatbot-unify branch